scripts/resolve-workspace-deps.mjs

```
@@ -0,0 +1,32 @@
+  for (const depType of ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies']) {
+    if (!pkg[depType]) continue;
+    for (const [dep, ver] of Object.entries(pkg[depType])) {
+      if (ver.startsWith('workspace:') && versions[dep]) {
+        pkg[depType][dep] = versions[dep];
+        changed = true;
+      }
+    }
+  }
+  if (changed) {
+    writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + '\n');
+    console.log(`Resolved workspace deps in ${pkgPath}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)